### PR TITLE
Training content

### PR DESCRIPTION
**Pull request by klieret**

### CODE REVIEW

File: _training/howto-new-module.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: Creating a HSF module from scratch
+layout: plain
+---
+
+If
+
+* You want to create a new carpentry-style training lesson from scratch (or from material that is in another format)
```
Comment (graeme-a-stewart) [inline _training/howto-new-module.md:8]: Stylisticlly a lower case "you" is more correct

File: _training/howto-new-module.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+* You have talked to us about it
```
Comment (graeme-a-stewart) [inline _training/howto-new-module.md:9]: Ditto

File: _training/howto-update-module-style.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+title: Updating a carpentry-style module with the HSF style
+layout: plain
+---
+
+If
```
Comment (graeme-a-stewart) [inline _training/howto-update-module-style.md:6]: "If..."

File: _training/howto-update-module-style.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+* You have built a carpentry style lesson by following the software carpentry tutorial
```
Comment (graeme-a-stewart) [inline _training/howto-update-module-style.md:8]: Lower case "you" (and in following lines)

File: _training/howto-update-module-style.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+4. Pull!
+
+   ```sh
+   git pull template gh-pages
+   ```
+
+5. Merge conflicts? Sorry to hear that. There's a couple of options:
+    1. Feeling lost? Abort by doing ``git reset --hard`` (note that all non-commited changes will be lost!)
+    2. Resolve the merge conflicts and commit! That would be awesome. If resolve by overwriting a file with with the version from the styles repository, ``git checkout template/gh-pages -- path/to/your/file`` is your friend
```
Comment (graeme-a-stewart) [inline _training/howto-update-module-style.md:47]: "resolve" -> "resolving"

File: _training/howto-new-module.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+then this is for you! If you already have a module of some kind and want to update it with the HSF style, see [here](howto-update-module-style).
+
+Creating the repository and getting started
+-------------------------------------------
+
+**We recommend you to let us do this part, so that you can start with the content right away!**
+
+Our training modules are mimicing the work of the [software carpentries](https://software-carpentry.org/), so we essentially follow their [guide](https://carpentries.github.io/lesson-example/) to set this up, but please be aware of the following changes:
+
+* Please use our version of the styles repository to get started, that is ``https://github.com/hsf-training/hsf-styles `` instead of ``https://github.com/carpentries/styles``
```
[the remark this answers is not in the document]
Comment (klieret) [reply _training/howto-new-module.md:20]: It's for importing the correct repository, so the link needs to be copied

**Comment by klieret** [comment]
Thanks a lot for all of your suggestions @graeme-a-stewart, I very much appreciate these! I've addressed all of them (except for replacing the URLs with links as noted).

File: _training/howto-new-module.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (graeme-a-stewart) [reply _training/howto-new-module.md:20]: Sure - fine then!

**Pull request merged**